### DISCUSSION TITLE
Wagtail GSoC'23

### DISCUSSION THREAD

**Discussion by all4one-max** [wagtail/wagtail#9834] [General]
hi! I came across wagtail while going through GSoC'22 organisations. I am interested in contributing to this organisation. Will wagtail be part of GSoC 2023?

**Comment by thibaudcolas** [comment]
Hi @all4one-max 👋. Good question, we’re currently wondering the same thing. Here is the [GSoC 2023 timeline](https://developers.google.com/open-source/gsoc/timeline). We’ll know whether Wagtail is applying to participate towards the end of January, and if we apply, and get accepted, this will be shared on February 22nd, 18:00 UTC.

I’d recommend joining our [Slack workspace](https://github.com/wagtail/wagtail/wiki/slack) if you want to hear the news as soon as possible, or if not, follow discussions here or Twitter [@wagtailcms](https://twitter.com/wagtailcms) or Mastodon [@wagtail@fosstodon.org](https://fosstodon.org/@wagtail).
